source/extensions/filters/http/set_metadata/config.cc

```diff
@@ -0,0 +1,34 @@
+#include "extensions/filters/http/set_metadata/config.h"
+
+#include <string>
+
+#include "envoy/extensions/filters/http/set_metadata/v3/set_metadata.pb.h"
+#include "envoy/extensions/filters/http/set_metadata/v3/set_metadata.pb.validate.h"
+#include "envoy/registry/registry.h"
+
+#include "common/protobuf/utility.h"
+
+#include "extensions/filters/http/set_metadata/set_metadata_filter.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace SetMetadataFilter {
+
+Http::FilterFactoryCb SetMetadataConfig::createFilterFactoryFromProtoTyped(
```

Comment:
Is there a test covering this?

Reply:
Added in https://github.com/envoyproxy/envoy/pull/16400/commits/cfad0846aa208507d3e5450807bda6512209028c (config_test.cc)